extensions/BugModal/web/comments.js

```diff
@@ -367,11 +360,11 @@ $(function() {
 
         // update Bugzilla
         try {
-            var result = await Bugzilla.API.put(
+            await Bugzilla.API.put(
               `bug/comment/${commentID}/tags`,
-              { add: newTags, comment_tag_html: 1 }
+              { add: newTags }
             );
-            renderTags(commentNo, result.comment_tag_html);
+            refreshTags(commentNo, commentID);
```

Comment:
I think it would be better to have the single internal bug-modal call return all the required tag info rather than making two calls, but would like to check the UX with two calls.

Please let me know when this change is on bugzilla-dev.